src/pyspiffe/workloadapi/default_jwt_source.py

```diff
@@ -0,0 +1,149 @@
+"""
+This module defines the default source implementation for JWT Bundles and SVIDs.
+"""
+import logging
+import threading
+from typing import Optional, Set
+
+from pyspiffe.spiffe_id.spiffe_id import SpiffeId
+from pyspiffe.bundle.jwt_bundle.jwt_bundle import JwtBundle
+from pyspiffe.bundle.jwt_bundle.jwt_bundle_set import JwtBundleSet
+from pyspiffe.spiffe_id.trust_domain import TrustDomain
+from pyspiffe.svid.jwt_svid import JwtSvid
+from pyspiffe.workloadapi.default_workload_api_client import DefaultWorkloadApiClient
+from pyspiffe.workloadapi.workload_api_client import WorkloadApiClient
+from pyspiffe.workloadapi.jwt_source import JwtSource
+from pyspiffe.workloadapi.exceptions import JwtSourceError
+
+
+class DefaultJwtSource(JwtSource):
+    """Default implementation of JwtSource. This class may be used by clients to get updated JWT-SVIDs and JWT bundles without the need to contact WorkloadAPI.
```

Comment:
This is true for the bundles, but for the JWT-SVID it will connect to the Workload API each time as the client may need a token with different audience.

Reply:
hmmmmm that is a good catch Max, thanks! I will make the change and make audience a method parameter.